runtime/moonbase/src/precompiles.rs

```diff
@@ -70,11 +71,17 @@ impl<R> PrecompileSet for MoonbasePrecompiles<R>
 where
 	R::Call: Dispatchable<PostInfo = PostDispatchInfo> + GetDispatchInfo + Decode,
 	<R::Call as Dispatchable>::Origin: From<Option<R::AccountId>>,
-	R: parachain_staking::Config + pallet_evm::Config + pallet_crowdloan_rewards::Config,
-	R::AccountId: From<H160>,
+	R: parachain_staking::Config
+		+ pallet_evm::Config
+		+ pallet_crowdloan_rewards::Config
+		+ pallet_democracy::Config,
+	R::AccountId: From<H160>, //TODO we shouldn't need this. There is a method for it in frontier.
```

Comment:
See this comment : https://github.com/PureStake/moonbeam/pull/518/files#r707395678